Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,293 @@
+configuration:
+- display: url
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: user
+  name: user
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: password
+  name: password
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: insecure
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+script:
+  script: |
+    # ''' IMPORTS '''
+    import json
+
+    import requests
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # ''' GLOBALS '''
+    USERNAME = demisto.params().get('user')
+    PASSWORD = demisto.params().get('password')
+    BASE_URL = demisto.params().get('url')
+    VERIFY_SSL = demisto.params().get('insecure')
+
+
+    # ''' HELPERS '''
+    def get_session():
+        session = requests.Session()
+        session.verify = VERIFY_SSL
+        return session
+
+
+    # ''' FUNCTIONS '''
+    @logger
+    def login():
+        return session.post(BASE_URL + '/api/login',
+                            json={'username': USERNAME,
+                                  'password': PASSWORD})
+
+
+    def logout():
+        session.post(BASE_URL + '/api/login/logout')
+
+
+    def fetch_incidents():
+        try:
+            r_events = session.post(BASE_URL + '/api/events',
+                                    json={"archive": False})
+            if r_events.status_code is not 200:
+                return_error('Error in API call. More information: {}, {}'.format(r_exclusion.status_code, r_exclusion.reason))
+            else:
+                incidents = []
+                for event in r_events.json():
+                    incident = {
+                        'name': event['type'],
+                        'rawJSON': json.dumps(event)
+                    }
+                    incidents.append(incident),
+                    session.put(BASE_URL + '/api/events/archive',
+                                json={'events': event['id']})
+                demisto.incidents(incidents)
+        except Exception, e:
+            return_error('Error while fetching incidents. More information: {}'.format(e.message))
+
+
+    @logger
+    def add_exclusion(exclusion_type, exclusion_data, exclusion_description, groups_id):
+        json_params = {'type': exclusion_type,
+                       'description': exclusion_description,
+                       'data': exclusion_data,
+                       'appliedGroupsIds': groups_id}
+        response = session.post(BASE_URL + '/api/exclusions',
+                                json=json_params)
+        if response.status_code is not 200:
+            return_error('Error while adding exclusion. More information: {}, {}'.format(response.status_code, response.reason))
+        demisto.results(response.json())
```

Comment:
The output needs to be an entry. See:
https://github.com/demisto/content/blob/master/docs/context_and_ouputs/README.MD
https://github.com/demisto/content/blob/master/docs/code_conventions/README.MD (`demisto.results()`)